src/qt/addressbookpage.cpp

```diff
@@ -281,7 +281,9 @@ void AddressBookPage::on_exportButton_clicked()
     // CSV is currently the only supported format
     QString filename = GUIUtil::getSaveFileName(this,
         tr("Export Address List"), QString(),
-        tr("Comma separated file", "Name of CSV file format") + QLatin1String(" (*.csv)"), nullptr);
+        /*: Expanded name of the CSV file format.
```

Comment:
How do we avoid having to translate this twice, same with PSBT?

Reply:
We code avoid code duplication by making constants. But not sure if it worth.